src/modules/account/views/ViewAccountAssets.vue

```diff
@@ -116,6 +119,49 @@ export default {
                 return false
             }
         }
+        const publicAccount = PublicAccount.createFromPublicKey(acc.value?acc.value.publicKey:'0'.repeat(64),AppState.networkType)
+        const noBalanceMosaics = ref([])
+        const fetchNoBalanceMosaics = async()=>{
+            if(!acc.value){
+                return 
+            }
+            let txnAPI = AppState.chainAPI.transactionAPI
+            let txnQueryParams = new TransactionQueryParams
+            txnQueryParams.embedded = true
+            txnQueryParams.type = [TransactionType.MOSAIC_DEFINITION]
+            txnQueryParams.publicKey = publicAccount
+            let searchedTxn = await txnAPI.searchTransactions(TransactionGroupType.CONFIRMED,txnQueryParams)
```

Comment:
I think it's better to use the new mosaic endpoint.  you can start testing the new endpoint at "api-1.testnet2.xpxsirius.io:3001".  The new rest will be deployed to testnet and mainnet soon